### PR TITLE
Added few items to "Java useful libs" list

### PR DESCRIPTION
**Pull request by RSheremeta**
Added:

- https://github.com/RSheremeta/TestRail-JUnit - This Library helps you to integrate your JUnit-based tests with TestRail in an easily manner. 
- https://github.com/RSheremeta/TestRail-TestNG - This Library helps you to integrate your TestNG-based tests with TestRail in an easily manner.
- https://github.com/RSheremeta/Allure-Environment-Creator - This tiny Library helps you to serve and store environment data for Allure easily.

### CODE REVIEW

**Comment by GastonAGenaud** [comment]
Nice!

File: java-test-automation.md [hunk 1]
```diff
@@ -222,6 +222,9 @@ Also:
 * [Truth](https://truth.dev/) - Fluent assertion framework for Java and Android from Google.
 * [JSONassert](https://github.com/skyscreamer/JSONassert) - JSON assertion library.
 * [selenium-to-restassured](https://github.com/mwinteringham/selenium-to-restassured) - Adapter to easily share Selenium WebDriver cookies with Rest Assured back and forth with this Java api.
+* [TestRail-JUnit](https://github.com/RSheremeta/TestRail-JUnit) - This Library helps you to integrate your JUnit-based tests with TestRail in an easily manner.
```
Comment (dc-avasilev) [inline java-test-automation.md:225]: plz, delete space in the end

**Review by dc-avasilev — CHANGES_REQUESTED** [review]
need some small fix

File: java-test-automation.md (hunk 1 above)
Comment (RSheremeta) [reply java-test-automation.md:225]: fixed

**Review by dc-avasilev — APPROVED** [review]

**Pull request merged**